### DISCUSSION TITLE
Add data set splitter tool

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5627] [Feature Requests]
Sp6 has a utility for splititng large data sets.

It would be trivial to implement the same in sp7.

It should either be an option during the import process (a checkbox `Split data set into smaller data sets of [numeric field] records?`) or a separate tool unrelated to the data set import view

**Comment by grantfitzsimmons** [comment]
Would be useful for large uploads, but maybe not as necessary if we implement #844
